adapters/eea-direct.js

```diff
@@ -0,0 +1,177 @@
+'use strict';
+import { acceptableParameters, convertUnits } from '../lib/utils';
+import { REQUEST_TIMEOUT } from '../lib/constants';
+import { default as baseRequest } from 'request';
+import { default as moment } from 'moment-timezone';
+import { parallel, map, filter } from 'async';
+import { parse } from 'babyparse';
+import uniqBy from 'lodash.uniqBy';
```

Comment:
Discussing this with @maxgrossman, this was added in Lodash 4. Currently checking if upgrading lodash will break anything. Separate PR forthcoming.